WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/creation/coupon/edit/OrderCreateCouponEditViewModel.kt

```diff
@@ -1,26 +1,22 @@
 package com.woocommerce.android.ui.orders.creation.coupon.edit
 
 import android.os.Parcelable
-import androidx.annotation.StringRes
 import androidx.lifecycle.SavedStateHandle
 import androidx.lifecycle.asLiveData
 import androidx.lifecycle.viewModelScope
-import com.woocommerce.android.R
 import com.woocommerce.android.extensions.isNotNullOrEmpty
-import com.woocommerce.android.ui.orders.creation.coupon.edit.CouponValidator.CouponValidationResult
 import com.woocommerce.android.viewmodel.MultiLiveEvent
 import com.woocommerce.android.viewmodel.ScopedViewModel
 import com.woocommerce.android.viewmodel.getNullableStateFlow
 import com.woocommerce.android.viewmodel.navArgs
 import dagger.hilt.android.lifecycle.HiltViewModel
 import kotlinx.coroutines.flow.combine
-import kotlinx.coroutines.flow.update
+import kotlinx.coroutines.flow.map
 import kotlinx.parcelize.Parcelize
 import javax.inject.Inject
 
 @HiltViewModel
 class OrderCreateCouponEditViewModel @Inject constructor(
```

Comment:
```suggestion
class OrderCreateCouponDetailsViewModel @Inject constructor(
```
Now coupon editing is not possible. What do you think about renaming the view model, fragment, and screen to "CouponDetails" instead of "CouponEdit"?

Reply:
Makes sense. Refactored here https://github.com/woocommerce/woocommerce-android/pull/9561/commits/c5d9a3cf66d67ad206360a44fab6d32f6500d19a